test/fixtures/loop_customtag.html

```diff
@@ -1,5 +1,8 @@
 <p>x</p>
```

Comment:
a failing test would be great if not already there. 
else , Looks good  👍

Reply:
failing test is in a7c6ef3d0aafc969284b9e96c5cd71bd54f22f70

Reply:
I meant test to show `the "${tagName}" tag must have a "loop" attribute` thing.

Reply:
ah, that was a bug introduced by this PR (in 9874a8d) - didn't add a failing test, just fixed it directly...

Reply:
ohh ok cool ,